tickets/src/main/java/no/unit/nva/publication/ticket/update/UpdateTicketHandler.java

```diff
@@ -77,31 +81,71 @@ public UpdateTicketHandler() {
 
     protected UpdateTicketHandler(TicketService ticketService, ResourceService resourceService, DoiClient doiClient,
                                   Environment environment) {
-        super(UpdateTicketRequest.class, environment);
+        super(TicketRequest.class, environment);
         this.ticketService = ticketService;
         this.resourceService = resourceService;
         this.doiClient = doiClient;
     }
 
     @Override
-    protected void validateRequest(UpdateTicketRequest updateTicketRequest, RequestInfo requestInfo, Context context)
+    protected void validateRequest(TicketRequest ticketRequest, RequestInfo requestInfo, Context context)
         throws ApiGatewayException {
         //Do nothing
     }
 
     @Override
-    protected Void processInput(UpdateTicketRequest input, RequestInfo requestInfo, Context context)
+    protected Void processInput(TicketRequest input, RequestInfo requestInfo, Context context)
         throws ApiGatewayException {
         var requestUtils = RequestUtils.fromRequestInfo(requestInfo);
         var ticket = fetchTicket(requestUtils);
-        if (hasEffectiveChanges(ticket, input)) {
-            updateTicket(input, requestUtils, ticket);
+
+        switch (input) {
+            case UpdateTicketRequest request -> handleUpdateTicketRequest(ticket, request, requestUtils);
+            case UpdateTicketOwnershipRequest request -> updateOwnership(ticket, request, requestUtils);
+            default -> {
+                // NO OP
+            }
```

Comment:
I think we simply ignore all we do not support. ApiGatewayHandler will by default throw BadRequest when it is failing on deserialization.